configs/lad/lad_r101_paa_r50_fpn_2x8_coco_1x.py

```diff
@@ -128,7 +128,3 @@
 
 # TODO: MMEngine does not support fp16 yet.
 # fp16 = dict(loss_scale=512.)
-# NOTE: `auto_scale_lr` is for automatically scaling LR,
```

Comment:
auto lr may need need to keep here, there isn't auto_scale_lr in _base_ configs

Reply:
auto_scale_lr is in in default_runtime.py